src/renderer/App.js

```diff
@@ -500,6 +507,10 @@ export default defineComponent({
       }
     },
 
+    setLocale: function() {
+      document.querySelector('html').setAttribute('lang', this.locale)
```

Comment:
Supported in every browser (e.g. Chrome v1, Firefox v1), and should be slightly faster than searching the DOM with `querySelector`: https://developer.mozilla.org/en-US/docs/Web/API/Document/documentElement

```suggestion
      document.documentElement.setAttribute('lang', this.locale)
```